assessment/showtest.php

```diff
@@ -2056,7 +2058,7 @@
 			if ($r=='success') {
 				echo '{success: true}';
 			} else {
-				echo '{error: "'.$r.'"}';
+				echo '{error: "$r"}';

```

Comment:
Since the $r is inside single quotes, this will prevent the variable interpolation from happening.